helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -14,68 +14,54 @@
-                      </tr>
-                      <!-- FOOTER -->
-                      <tr>
-                        <td align="center" style="min-width: 590px;">
-                          <table width="590" border="0" cellpadding="0" bgcolor="#875A7B" style="min-width: 590px; background-color: rgb(135,90,123); padding: 20px; border-collapse:separate;">
-                            <tr>
-                              <td valign="middle" align="left" style="color: #fff; padding-top: 10px; padding-bottom: 10px; font-size: 12px;">
-                                ${object.company_id.phone}
-                              </td>
-                              <td valign="middle" align="left" style="color: #fff; padding-top: 10px; padding-bottom: 10px; font-size: 12px;">
-                                ${object.company_id.email}
-                              </td>
-                              <td valign="middle" align="left" style="color: #fff; padding-top: 10px; padding-bottom: 10px; font-size: 12px;">
-                                ${object.company_id.website}
-                              </td>
-                            </tr>
-                          </table>
-                        </td>
-                      </tr>
-                    </tbody>
-                </table>
-                </body>
-                </html>
+            ><![CDATA[
+                <div>
+                    <p>Hello ${object.partner_id.name or object.partner_name or ''},</p>
+                    <p>The ticket "${object.display_name or 'n/a'}" has been closed.</p>
+                    % if object.partner_can_access():
+                        <div style="margin: 16px 0px 16px 0px;">
+                            <a href="${object.get_access_link()}" style="background-color: #875A7B; padding: 8px 16px 8px 16px; text-decoration: none; color: #fff; border-radius: 5px; font-size:13px;">
+                                View Ticket
+                            </a>
+                        </div>
+                    % endif
+                    Thank you,<br/><br/>
+                    ${user.signature | safe}<br/>
+                </div>
             ]]></field>
         </record>
+        <record id="created_ticket_template" model="mail.template">
+            <field name="name">Helpdesk Created Ticket Notification Email</field>
+            <field name="model_id" ref="helpdesk_mgmt.model_helpdesk_ticket" />
+            <field name="email_from">${object.company_id.partner_id.email}</field>
+            <field
+                name="email_to"
+            >${('' if object.partner_id.email and object.partner_id.email == object.partner_email else object.email) | safe},</field>
+            <field name="subject">The ticket ${object.number} has been created.</field>
+            <field
+                name="partner_to"
+            >${object.partner_id.id if object.partner_id.email and object.partner_id.email == object.partner_email else False}</field>
```

Comment:
**question**: So the way this works is `partner_to` gets set (and `email_to` not) if the partner's email has a value and vice versa, right?

I'm asking to make sure I understood correctly.

Reply:
Indeed, it mimic the way it is done in official `calendar` module
(https://github.com/odoo/odoo/blob/71db47cff99ce3b2090c3be5dfa3205654625d51/addons/calendar/data/mail_template_data.xml#L9-L10)

Reply:
Thanks.

Fine by me then :)